docs/high_level_design.adoc

```diff
@@ -1,9 +1,13 @@
 = Cloud Fabric Alcor Control Agent
+Eric Li <sze.li@futurewei.com>
+v0.2, 2020-03-01
+:toc: right
 
 == Summary
 
-The Cloud Fabric Alcor Control Agent (ACA) runs on each host machine. It serves as a stateless proxy between network controller and host machine networking components for control plane operations.
+The Cloud Fabric Alcor Control Agent (ACA) runs on each host machine. It serves as a stateless proxy between Alcor controller and host machine networking components for control plane operations.
 
+image::images/Agent_components.JPG["Component diagram", width=800, link="images/Agent_components.JPG"]

```

Comment:
This image somehow doesn't show up.

Reply:
I think because the image links are based on relative paths, so it won't show up correctly until the code is merged.